### ISSUE TITLE
Documentation needs fixed for passing keys as parameter in db.view

### ISSUE THREAD

**Issue by PaulAllan1** [dscape/nano#304]
Currently the documentation for db.view is as follows:
"db.view(designname, viewname, [params], [callback])

calls a view of the specified design with optional query string additions params. if you're looking to filter the view results by key(s) pass an array of keys, e.g { keys: ['key1', 'key2', 'key_n'] }, as params."

In actual fact what needs to be passed in to filter by keys is more like {keys: [['key1', 'key2', 'key_n']] }. Please update the documentation.

**Comment by franklinlindemberg** [comment]
I agree with @PaulAllan1, please update the documentation. I lost much time trying to figure it out.
Regards.

**Comment by carlosduclos** [comment]
This repository has been merged into apache/couchdb-nano, please continue the discussion [here](https://github.com/apache/couchdb-nano/issues/17).
